delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/stream/Supervisor.scala

```diff
@@ -105,23 +117,33 @@ object Supervisor {
     stop = Task.unit
   )
 
+  private[sourcing] val watchRestartMetadata = ProjectionMetadata("system", "watch-restarts", None, None)
```

Comment:
Should `DeleteExpired`, and `PurgeElemFailures` be integrated inside the supervisor in the future?

Reply:
`PurgeElemFailures` may fall into into it yes.
The idea was that at least the watch-restarts part is mandatory for the supervisor to work properly and I wanted to include it within the supervisor creation itself and not to rely distage magic to assemble it at startup.
It seemed more atomic and more understable that way, what do you think ?

Reply:
I agree, but as I wrote the comment I was just thinking that if other projections get added like this it might be putting to much in the supervisor.

Reply:
Yes, I think the ones to be included here must be related to the supervisor.